service/src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -1074,6 +1120,19 @@ public ResponseEntity<ApiGcpDataprocClusterResource> getDataprocCluster(
             .validateControlledResourceAndAction(
                 userRequest, workspaceUuid, resourceUuid, SamControlledResourceActions.READ_ACTION)
             .castByEnum(WsmResourceType.CONTROLLED_GCP_DATAPROC_CLUSTER);
+
+    DataprocCow dataprocCow = crlService.getDataprocCow();
+    ClusterName clusterName = resource.toClusterName();
+    try {
+      Cluster retrievedCluster = dataprocCow.clusters().get(clusterName).execute();
+      Gson gson = new GsonBuilder().setPrettyPrinting().create();
+      String prettyJsonString = gson.toJson(JsonParser.parseString(retrievedCluster.toString()));
+      System.out.println(prettyJsonString);
+
+    } catch (IOException e) {
+      System.out.println("Error retrieving cluster");
+    }
```

Comment:
Is this leftover from debugging or intended to log cluster state after the update? If this is intentional, it should use `logger` instead

Reply:
Oh, this is leftover from debugging.. thanks.